src/styles/blocks/header.scss

```diff
@@ -0,0 +1,80 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+
+  * {
+    justify-content: space-between;
+    background-color: white;
+    width: 100%;
+    box-sizing: border-box;
```

Comment:
create a selector at the top level and move this rule there
```
* {
  box-sizing: border-box;
}
```

Reply:
moved it to the badi, okay?